consultation_analyser/support_console/jinja2/support_console/consultations/export_audit.html

```diff
@@ -10,16 +11,27 @@ <h1 class="govuk-heading-l">{{ page_title }}</h1>
 <form method="post" enctype="multipart/form-data" novalidate>{{ csrf_input }}
 
   <div class="govuk-form-group">
+    {{ govukCheckboxes({
+      'name': "question_parts",
+      'fieldset': {
+        'legend': {
+          'text': "Which questions do you want to download data for?",
+          'isPageHeading': false,
+          'classes': "govuk-fieldset__legend--l"
+        }
+      },
+      'items': question_parts_items
```

Comment:
I wonder if this is going to be annoying to navigate in consultations with lots of questions with long titles - something to keep an eye on (and check with users) in the future

Reply:
Yep - good point. I think one to worry about in the future (if and when it becomes a problem).